DiscordBot/Program.cs

```diff
@@ -49,9 +49,9 @@ public async Task StartAsync(string[] args)
             await _client.StartAsync();
             _handler = new CommandHandler();
             await _handler.InitializeAsync(_client);
+            while (_client.ConnectionState != ConnectionState.Connected) { }
```

Comment:
Replaces Task.Delay(), making the program update roles once the bot connects to the server.